python/src/genetic_search.py

```diff
@@ -274,9 +293,13 @@ def maybe_add_child_to_offspring(
             offspring[lowest_cost_parent] = mating_pool[lowest_cost_parent]
 
             # perform a local search on best individual as well, don't replace the one we just added
-            valid_indices = [idx for idx in range(self._num_individuals) if idx != lowest_cost_parent]
+            valid_indices = [
```

Comment:
is ur auto formatter thing configured to 80 characters per line? Would you mind bumping that up to 120, i think thats what pylint is set to. Would get rid of a bunch of these type of changes i think

Reply:
I am not sure, I will look into this later